objects/namespace-install.yaml

```diff
@@ -1763,6 +1808,23 @@ spec:
                 - kind
                 type: object
               type: array
+            namespaceResourceWhitelist:
+              description: NamespaceResourceWhitelist contains list of whitelisted
+                namespace level resources
+              items:
+                description: GroupKind specifies a Group and a Kind, but does not
+                  force a version.  This is useful for identifying concepts during
```

Comment:
`  This is` Extra whitespace

Reply:
this is from upstream, I'd rather not touch it as we're going to move to kustomize anyway, you should bring it up with the argo-cd folks :)